setup.py

```diff
@@ -57,36 +57,31 @@ def run_tests(self):
     install_requires=[
         'pyyaml',
         'requests',
-        'numpy>=1.20.0',
-        'pandas>=1.0.0,!=1.0.4',
+        'numpy',
+        'pandas',
         'joblib',
-        'pyarrow>=3.0.0',
+        'pyarrow',
         'dask[complete]>=2022.10.0',
         'docker',
-        'boto3>=1.10.44',
-        's3fs>=0.4.0,<0.5.0',
-        'fsspec>=0.6.0',
-        'docutils<0.15,>=0.10',
-        'yamale>=2.0',
-        'ruamel.yaml>=0.15.0',
-        'testfixtures',
+        's3fs[boto3]',
```

Comment:
We had to pin s3fs to a really old version so we could have boto3 and s3fs coexist. Turns out [we weren't the only ones with this problem](https://github.com/fsspec/s3fs/issues/357) so they made a way to resolve the conflict, which we're using now. This allows us to use a newer version of s3fs among other things.